frontend/src/scenes/sceneTypes.ts

```diff
@@ -53,7 +53,7 @@ export enum Scene {
     AsyncMigrations = 'AsyncMigrations',
     DeadLetterQueue = 'DeadLetterQueue',
     Billing = 'Billing',
-    Apps = 'Apps',
+    Apps = 'Connectors',
```

Comment:
This is why the E2E test is failing, what was `apps` is now `connectors`. In principle, all of these values should match the keys.
```suggestion
    Apps = 'Apps',
```